src/index.ts

```diff
@@ -264,10 +292,11 @@ export default class Wallet {
    */
   public static generate(icapDirect: boolean = false): Wallet {
     if (icapDirect) {
-      const max = new BN('088f924eeceeda7fe92e1f5b0fffffffffffffff', 16)
-      while (true) {
+      const max = BigInt('0x088f924eeceeda7fe92e1f5b0fffffffffffffff')
+      for (;;) {
```

Comment:
What is the difference here between this `while` and `for`?

Reply:
some linter rule required it, no difference